Sources/Array+CalculateMD5Hash.swift

```diff
@@ -1,20 +1,10 @@
 import Foundation
-import CommonCrypto
 
 extension Array where Element == String {
     func calculateMD5Hash() -> String? {
         let sortedArray = self.sorted()
         let elementsJoined = String(sortedArray.joined(separator: ","))
 
-        guard let messageData = elementsJoined.data(using: .utf8) else { return nil }
-        var digestData = Data(count: Int(CC_MD5_DIGEST_LENGTH))
-
-        _ = digestData.withUnsafeMutableBytes {digestBytes in
-            messageData.withUnsafeBytes {messageBytes in
-                CC_MD5(messageBytes, CC_LONG(messageData.count), digestBytes)
-            }
-        }
-
-        return digestData.map { String(format: "%02hhx", $0) }.joined()
+        return MD5(elementsJoined)
```

Comment:
So, can this MD5 implementation fail?

Can this function signature instead be `calculateMD5Hash() -> String`?

Reply:
>So, can this MD5 implementation fail?

There are a few tests that validate [implementation](https://github.com/onmyway133/SwiftHash/blob/master/SwiftHashTests/Tests.swift) of MD5. I don't have exact answer because I didn't test it throughoutly for every single scenario/edge case that could potentially happen. It's the same case with all third party dependencies ... Ideally we would use [CommonCrypto](https://opensource.apple.com/source/CommonCrypto/) from Apple but there's a lot of [issues](https://stackoverflow.com/questions/25248598/importing-commoncrypto-in-a-swift-framework) and no elegant solutions in order to make it work in a Swift framework (this was resolved in Xcode 10 but we still need to support Xcode 9).

>Can this function signature instead be calculateMD5Hash() -> String?

In `calculateMD5Hash` function I'm also first sorting the elements in the array and then joining them together in order to calculate the MD5 hash. By using just `MD5` function I would need to move this logic elsewhere and I don't see a point of doing that.